edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -644,28 +652,30 @@ impl ModuleRuntime for DockerModuleRuntime {
         #[cfg(windows)]
         let uptime: u64 = unsafe { winapi::um::sysinfoapi::GetTickCount64() / 1000 };
 
-        let mut system_info = sysinfo::System::new();
-        system_info.refresh_all();
+        let mut system_resources = self.system_resources.as_ref().lock().unwrap();
```

Comment:
Would there be any perf. side impact?

Reply:
There shouldn't be. This is the only consumer of the lock, its only needed b/c the runtime is clone.